src/exampleTypes/TermsStatsTable.js

```diff
@@ -80,7 +80,10 @@ let TermsStatsTable = injectTreeNode(
                                   tree.getNode(criteria).children
                                 )
 
-                              if (!filter) {
+                              if (
+                                !filter ||
+                                _.get('mode', filter) === 'exclude'
```

Comment:
We should probably model this similar to `InteractionBehaviors` from main app. To filter by an item on facet, you add it if it's not present on include and remove it if it is present on exclude. Our pattern thus far has been to expose `contexture-client` actions for most of InteractionBehaviors, we just haven't added any type-specific ones yet.

@michsa might have some thoughts, too